File: PronoSpritzWeb/src/app/competition/competition/match.dto.ts
// export interface MatchDto {
//   // teamImageUrl: string;
//   teamName: string;
//   liveScore: string;
//   predictionScore: string;
//   status: string;
// }

export interface MatchDto {
    Id: number;
    CompetitionId: number;
    StartTime: Date | string;
    Status: string;
    MatchDay: number;
    Group: string;
    HomeTeam: string;
    HomeTeamId: number;
    AwayTeam: string;
    AwayTeamId: number;
    HomeGoals: number;
    AwayGoals: number;
}

File: PronoSpritzApi/PronoSpritzApi/Models/Football/MatchDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PronoSpritzApi.Models.Football
{
    public class MatchDto
    {
        public int Id { get; set; }
        public int CompetitionId { get; set; }
        public DateTime StartTime { get; set; }
        public string Status { get; set; }
        public int MatchDay { get; set; }
        public string Group { get; set; }
        public string HomeTeam { get; set; }
        public int HomeTeamId { get; set; }
        public string AwayTeam { get; set; }
        public int AwayTeamId { get; set; }
        public int HomeGoals { get; set; }
        public int AwayGoals { get; set; }
    }
}   